eng/common/scripts/Package-Properties.ps1

```diff
@@ -243,20 +336,12 @@ function Get-PrPkgProperties([string]$InputDiffJson) {
             # handle changes to files that are RELATED to each package
             foreach($triggerPath in $triggeringPaths) {
                 $resolvedRelativePath = (Join-Path $RepoRoot $triggerPath)
-                if (!$triggerPath.StartsWith("/")){
-                    $resolvedRelativePath = (Join-Path $RepoRoot "sdk" "$($pkg.ServiceDirectory)" $triggerPath)
-                }
-
-                # if we are including this package due to one of its additional trigger paths, we need
-                # to ensure we're counting it as included for validation, not as an actual package change
-                if ($resolvedRelativePath) {
-                    $includedForValidation = $filePath -like (Join-Path "$resolvedRelativePath" "*")
-                    $shouldInclude = $shouldInclude -or $includedForValidation
-                    if ($includedForValidation) {
-                        $pkg.IncludedForValidation = $true
-                    }
-                    break
+                $includedForValidation = $filePath -like (Join-Path "$resolvedRelativePath" "*")
```

Comment:
This is now completed when resolving the artifact to begin with. Users can place their relative paths in their `ci.yml`, but we will only ever see the full format EG:

within `sdk/eventhub/ci.yml`
```yml
- artifactname
  triggeringPaths:
    - Azure.Messaging.EventHubs.Shared
```

becomes 

```
sdk/eventhub/Azure.Messaging.EventHubs.Shared`
```